mini-projects/00-jq-rs/Cargo.toml

```diff
@@ -4,3 +4,9 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
+
+anyhow = "1.0.86"
+clap = { version = "4.5.20", features = ["derive"] }
+colored = "2.1.0"
+regex = "1.10.6"
+serde_json = "1.0.128"
```

Comment:
nit: you could use the `preserve_order` feature to preserve the original order of the json when `sort-keys` isn't specified